packages/core/upload/admin/src/components/AssetTableList/PreviewCell.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { prefixFileUrlWithBackendUrl, pxToRem } from '@strapi/helper-plugin';
+import { Avatar } from '@strapi/design-system/Avatar';
+import { Flex } from '@strapi/design-system/Flex';
+import { Typography } from '@strapi/design-system/Typography';
+
+const FileWrapper = styled(Flex)`
+  border-radius: 50%;
+
+  span {
+    /* The smallest fontSize in the DS is not small enough in this case */
+    font-size: ${pxToRem(10)};
+  }
+`;
+
+export const PreviewCell = ({ alternativeText, fileExtension, mime, name, thumbnailURL, url }) => {
+  if (mime.includes('image')) {
+    const mediaURL = prefixFileUrlWithBackendUrl(thumbnailURL) ?? prefixFileUrlWithBackendUrl(url);
+
+    return <Avatar src={mediaURL} alt={alternativeText ?? name} preview />;
```

Comment:
```suggestion
    return <Avatar src={mediaURL} alt={alternativeText} preview />;
```

I'd propose to remove the `name` fallback: we have just removed that from the upload because it creates more harm than good. Screenreaders will automatically read the filename if no alt attribute is provided.

Reply:
`Avatar` doesn't seem to like not having an alt - does it mean it shouldn't be required in the DS? wdyt? @gu-stav

Reply:
Yes, I'd say so!

Reply:
On a feature branch you can merge whatever you want :)